package.json

```diff
@@ -0,0 +1,150 @@
+    "npm-run-all": "^4.1.5",
+    "nyc": "^14.1.1",
+    "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "postcss-loader": "^3.0.0",
+    "prettier": "^1.17.1",
+    "react-testing-library": "^7.0.1",
+    "redux-mock-store": "^1.5.3",
+    "sass-loader": "^7.1.0",
+    "stylelint": "^10.0.1",
+    "stylelint-config-recommended-scss": "^3.3.0",
+    "stylelint-scss": "^3.6.1",
+    "url-loader": "^1.1.2",
+    "webpack": "^4.32.2",
+    "webpack-cli": "^3.3.2",
+    "webpack-dev-server": "^3.4.1",
+    "webpack-merge": "^4.2.1"
+  },
+  "dependencies": {
+    "@salesforce-ux/design-system": "2.8.3",
+    "@salesforce/design-system-react": "^0.10.4",
+    "classnames": "^2.2.6",
+    "i18next": "^15.1.3",
+    "i18next-browser-languagedetector": "^3.0.1",
+    "i18next-xhr-backend": "^2.0.1",
+    "js-cookie": "^2.2.0",
+    "raven-js": "^3.27.1",
+    "react": "^16.8.6",
+    "react-document-title": "^2.0.3",
+    "react-dom": "^16.8.6",
+    "react-i18next": "^10.10.0",
+    "react-redux": "^7.0.3",
+    "react-router-dom": "5.0.0",
+    "redux": "^4.0.1",
+    "redux-devtools-extension": "^2.13.8",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.3.0",
+    "sockette": "^2.0.5",
+    "whatwg-fetch": "^3.0.0"
+  },
+  "scripts": {
+    "webpack:serve": "webpack-dev-server --config webpack.dev.js",
+    "django:serve": "python manage.py runserver",
+    "redis:clear": "redis-cli FLUSHALL",
+    "worker:serve": "python manage.py rqworker default short",
+    "scheduler:serve": "python manage.py rqscheduler default short",
+    "rq:serve": "npm-run-all redis:clear -p worker:serve scheduler:serve",
+    "serve": "run-p django:serve webpack:serve rq:serve",
+    "prettier": "prettier --write '*.js' '.*.js' 'src/js/*.js' 'src/js/**/*.js' 'test/js/*.js' 'test/js/**/*.js' 'src/sass/**/*.scss'",
+    "eslint": "eslint '*.js' 'src/js/*.js' 'src/js/**/*.js' 'test/js/*.js' 'test/js/**/*.js'",
+    "flow": "flow",
+    "stylelint": "stylelint 'src/sass/**/*.scss'",
+    "flake8": "flake8 manage.py metashare/ config/",
+    "isort": "isort -rc manage.py metashare/ config/",
+    "black": "black manage.py metashare/ config/",
+    "lintpy": "run-s isort black flake8",
+    "lintjs": "run-s prettier eslint",
+    "lint": "npm-run-all -p lintpy lintjs -s stylelint flow",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "JEST_JUNIT_OUTPUT='./coverage/junit.xml' jest --ci --runInBand --reporters=default --reporters=jest-junit",
+    "test:report-coverage": "cat ./coverage/lcov.info | coveralls",
+    "test:check-coverage": "nyc check-coverage --temp-directory coverage",
+    "pytest": "pytest",
+    "pytest:report-coverage": "$VIRTUAL_ENV/bin/coveralls",
+    "pytest:check-coverage": "coverage3 report --fail-under=100",
+    "build": "webpack --config webpack.dev.js",
+    "prod": "webpack --config webpack.prod.js",
+    "heroku-postbuild": "yarn prod"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
```

Comment:
This seems to be overridden by the Docker-specific pre-commit hook, so it breaks if Docker is not setup.

Reply:
Hm, since the `githooks` Docker changes the surrounding dev env, I am not sure what the best way around that might be. I'll consider it.

Reply:
Maybe we should discuss -- I'm not clear what the dedicated `githooks` container is doing. It seems worst case we can ditch lint-staged and just modify the Docker githook to check whether Docker is running?

Reply:
Let's discuss, yeah. It's doing something very different from what I think you think it's doing.

Reply:
I don't think I'm far off, but thanks. :) It's copying the hook from `hooks` into the local `.git/hooks` directory, no? Which replaces the husky `pre-commit` hook...

In which case I guess this is only a problem for users who have already run the Docker container... so let's just leave it alone. 👍

Reply:
Right; that container has to run exactly once, and it modifies the host environment. So when you said "check whether Docker is running" I thought you thought it was more of a continuous process. But yeah, it's just a one-off "get this hook symlinked" thing, if you run the Docker stuff. If you never do, it'll remain untouched.